runtimes/spiritnet/src/lib.rs

```diff
@@ -874,6 +890,7 @@ construct_runtime! {
 		// placeholder: parachain council election = 33,
 		TechnicalMembership: pallet_membership::<Instance1> = 34,
 		Treasury: pallet_treasury = 35,
+		DynFilter: pallet_dyn_filter = 36,
```

Comment:
Wouldn't we rather want to move this to the section of KILT Pallets?

Reply:
I thoughts it's part of the governance that's way i added it there. Maybe we even want to keep it (only callable by democracy). Just in case.

Reply:
Makes sense!